packages/components/src/ui/styles/theme/utils.js

```diff
@@ -0,0 +1,122 @@
+/**
+ * External dependencies
+ */
+/**
+ * Internal dependencies
+ */
+import { get } from '../../create-styles';
+import colorize from 'tinycolor2';
+
+/**
+ * @param {number} value
+ * @return {string} Space.
+ */
+export function space( value ) {
+	return `calc(${ get( 'gridBase' ) } * ${ value })`;
+}
+
+/**
+ * @param {Record<string, import('tinycolor2').ColorInput>} colors
+ * @return {Record<string, import('tinycolor2').ColorInput>} Text colors.
```

Comment:
I appreciate the types! Did you know that you can use an `@typedef` in JSDoc comments to avoid having to type the import every time? It's still a little awkward, but might make the types a tad more readable since you use it a lot.

```js
/**
 * @typedef {import('tinycolor2').ColorInput} ColorInput
 */
```

Then you can use `ColorInput` instead of typing out the whole thing.

Reply:
Yes, but that does have a side effect of polluting the exported types (all `@typedef`s are exported).

Reply:
😒 That's rather annoying, isn't it? Anyway, TIL, thanks!

Reply:
> Yes, but that does have a side effect of polluting the exported types (all @typedefs are exported).

That would be why I see the same type multiple times in some instances. That's unexpected but it makes sense.